Selenium.Axe/Selenium.Axe/AxeBuilder.cs

```diff
@@ -11,18 +14,82 @@ namespace Selenium.Axe
     public class AxeBuilder
     {
         private readonly IWebDriver _webDriver;
-        private readonly IncludeExcludeManager _includeExcludeManager = new IncludeExcludeManager();
 
-        private static readonly AxeBuilderOptions DefaultOptions = new AxeBuilderOptions {ScriptProvider = new EmbeddedResourceAxeProvider()};
+        private readonly AxeRunContext runContext = new AxeRunContext();
+        private readonly AxeRunOptions runOptions = new AxeRunOptions();
+        private static readonly AxeBuilderOptions DefaultOptions = new AxeBuilderOptions { ScriptProvider = new EmbeddedResourceAxeProvider() };
 
+        private static readonly JsonSerializerSettings JsonSerializerSettings = new JsonSerializerSettings
+        {
+            Formatting = Formatting.None,
+            NullValueHandling = NullValueHandling.Include
+        };
+
+        [Obsolete("Options is deprecated. This will be removed in future versions. Please use WithTags, WithRules & DisableRules apis instead")]
```

Comment:
There are other options axe-core supports besides rules/runOnly that this would currently allow settings; are there any others we want to implement first class support for before we mark this as obsolete?

Reply:
sure. i will send another PR to support that with WithOptions api which is strongly typed

---
In reply to: [311819014](https://github.com/devaradhanm/SeleniumAxeDotnet/pull/1#discussion_r311819014) [](ancestors = 311819014)